api/components/TopQuestions.tsx

```diff
@@ -0,0 +1,51 @@
+/** @jsxImportSource @hono/hono/jsx */
+import { type FC } from "@hono/hono/jsx";
+import { Question } from "../models/questions.ts";
+import { ArrowUpIcon } from "./assets/arrow-up.ts";
+
+const TopQuestions: FC<{ questions: Question[] }> = ({ questions }) => {
+  // QUESTION:
+  // 1. do we want questions to be scrollable?  currently - `overflow-y: scroll` ;
+  // 2. What if the length of the question requires line breaks?
+  // TODO: remove
+  questions = [...questions, ...questions, ...questions, ...questions];
```

Comment:
Why are we doing this?

Reply:
Testing purpose only. So a reviewer can check scrolling, resizing and fitting. Will remove before merge.